homeassistant/components/netatmo/select.py

```diff
@@ -42,39 +41,34 @@ def _create_entity(netatmo_home: NetatmoHome) -> None:
 class NetatmoScheduleSelect(NetatmoBaseEntity, SelectEntity):
     """Representation a Netatmo thermostat schedule selector."""
 
-    def __init__(
-        self,
-        netatmo_home: NetatmoHome,
-    ) -> None:
+    _attr_name = None
+
+    def __init__(self, netatmo_home: NetatmoHome) -> None:
         """Initialize the select entity."""
-        SelectEntity.__init__(self)
         super().__init__(netatmo_home.data_handler)
 
-        self._home = netatmo_home.home
-        self._home_id = self._home.entity_id
+        self.home = netatmo_home.home
 
-        self._signal_name = netatmo_home.signal_name
         self._publishers.extend(
             [
                 {
                     "name": HOME,
-                    "home_id": self._home.entity_id,
-                    SIGNAL_NAME: self._signal_name,
+                    "home_id": self.home.entity_id,
+                    SIGNAL_NAME: netatmo_home.signal_name,
                 },
             ]
         )
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, self.home.entity_id)},
+            name=self.home.name,
+            configuration_url=CONF_URL_ENERGY,
```

Comment:
Let's at least not leave it unknown
```suggestion
            name=self.home.name,
            manufacturer="Netatmo",
            model="Climate",
            configuration_url=CONF_URL_ENERGY,
```